unstract/flags/src/unstract/flags/clients/evaluation_client.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+import grpc
+
+from ..generated import evaluation_pb2, evaluation_pb2_grpc
+from .base_client import BaseClient
+
+
+class EvaluationClient(BaseClient):
```

Comment:
@athul-rs Good to add a doc string mentioning the purpose and/or how this client is different from others.